features/car/access.feature

```diff
@@ -50,14 +50,14 @@ Feature: Car - Restricted access
             | no     |                | yes             |          |                  |                   |      | x     |
             | no     |                | no              | yes      |                  |                   | x    | x     |
             | no     |                | no              | no       |                  | yes               |      | x     |
-            |        | no             |                 |          | no               |                   |      | x     |
-            |        |                | no              |          |                  | no                | x    |       |
+            |        | no             |                 |          | no               |                   |      |       |
```

Comment:
thanks for clarification! i changed behavior in this case: if access in one direction is specified but in other direction is not and can not be derived from the way type then set mode to `inaccessible`. This keeps old behavior for the test.